cli/src/offline/blockhash_query.rs

```diff
@@ -0,0 +1,394 @@
+use super::*;
+
+#[derive(Debug, PartialEq)]
+pub enum Source {
+    Cluster,
+    NonceAccount(Pubkey),
+}
+
+impl Source {
+    pub fn get_blockhash_fee_calculator(
```

Comment:
This method name reads to me like "get the fee calculator for this blockhash" which is what the other method does.

Reply:
That's a legacy method name, but I can toss an "and" in there, sure

Reply:
https://github.com/solana-labs/solana/pull/8750/commits/48b274dd6dc816a79ee51d45c433d094f3afecb8